app/controllers/outfits_controller.rb

```diff
@@ -11,12 +11,12 @@ def new
 
   def create
     @outfit = Outfit.new(outfit_params)
-    @user = User.find(params[:user_id])
+    @user = current_user
     @outfit.user = @user
     authorize @outfit
 
     if @outfit.save
-      redirect_to @outfit, notice: 'Your outfit was succesfully created'
+      redirect_to outfits_path, notice: 'Your outfit was succesfully created'
```

Comment:
should we re-direct to all outfits or only the show page of the outfit the user has created?

Reply:
oh I just changed it to outfit_path(@outfit) ;)